frontend/ui/uimanager.lua

```diff
@@ -735,16 +738,9 @@ function UIManager:quit(exit_code)
     self._task_queue_dirty = false
     self._running = false
     self._run_forever = nil
-    for i = #self._window_stack, 1, -1 do
-        table.remove(self._window_stack, i)
-    end
-    for i = #self._task_queue, 1, -1 do
-        table.remove(self._task_queue, i)
-    end
-    for i = #self._zeromqs, 1, -1 do
-        self._zeromqs[i]:stop()
```

Comment:
Dropped this along the way ;).

Reply:
Thanks very much here too, I really need new contact lenses.